keyboards/system76/launch_1/rules.mk

```diff
@@ -1,33 +1,23 @@
-# MCU name
-MCU = atmega32u4
+# AVR MCU options
+MCU = atmega32u4# MCU name
+BOOTLOADER = atmel-dfu# Bootloader selection
+F_CPU = 8000000# CPU frequency divided by two since AVR is at 3.3 V
+F_USB = 16000000# External oscillator is 16 MHz
 
-# CPU frequency divided by two since AVR is at 3.3 V
-F_CPU = 8000000
-
-# External oscillator is 16 MHz
-F_USB = 16000000
-
-# Bootloader selection
-BOOTLOADER = atmel-dfu
-
-# Build options
-#   change yes to no to disable
-BOOTMAGIC_ENABLE = no       # Bootmagic Lite
+# Feature options
+BOOTMAGIC_ENABLE = yes      # Bootmagic Lite
 MOUSEKEY_ENABLE = no        # Mouse keys
 EXTRAKEY_ENABLE = yes       # Audio control and system control
 CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
 DYNAMIC_KEYMAP_ENABLE = yes # Reconfigurable keyboard without flashing firmware
 NKRO_ENABLE = yes           # USB N-key rollover
 RAW_ENABLE = yes            # Raw HID commands (used by Keyboard Configurator)
```

Comment:
```suggestion
RAW_ENABLE = yes            # Raw HID commands (used by Keyboard Configurator)
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

Reply:
Also added the extra knobs as requested, but don't understand why as these 3 are explicitly set to "no" in the common features Makefile. The `show_full_features` target returns:
```text
Hardware Options:
  AUDIO_ENABLE                   = no               # file
  BACKLIGHT_ENABLE               = no               # file
  RGBLIGHT_ENABLE                = no               # file
```